content/200-concepts/300-more/050-environment-variables/200-using-multiple-env-files.mdx

```diff
@@ -7,38 +7,75 @@ tocDepth: 2
 
 <TopBlock>
 
-There is a risk that your production database could be deleted if you store different connection URLs to each of your environments within a single `.env` file.
+When developing an application you will generally have different environments such as `development`, `test` and `production` which you use to **separate the stages of the development cycle**.
 
-One solution is to have multiple `.env` files which each represent different environments. In practice this means you create a file for each of your environments:
+Prisma can access each environment via a [connection url](../../../reference/database-reference/connection-urls) which is stored in a `.env` file.
+
+</TopBlock>
+
+## Scenario: Running integration tests
+
+You have three [connection urls](../../../reference/database-reference/connection-urls) for each of your environments stored in a single `.env` file, each one connects to a different Postgres database.
+
+```env file=.env
+
+// Connection URL to development environment
+DEVELOPMENT_DATABASE_URL="postgresql://prisma:prisma@localhost:5432/development"
+
+// Connection URL to testing environment
+TESTING_DATABASE_URL="postgresql://prisma:prisma@localhost:5432/test"
+
+// Connection URL to production environment
+PRODUCTION_DATABASE_URL="postgresql://prisma:prisma@pg-prod.xxxx.rds.amazonaws.com.com:5432/production"
+
+```
+
+<Admonition type="warning">
+
+`PRODUCTION_DATABASE_URL` is added above for clarity in explaining the scenario. It's **not recommended** to store your production credentials locally, even if they are git ignored.
```

Comment:
I don't think _any_ of this recommended in any way! That would mean you need to update your schema file each time, regenerating the Client - you would get this wrong 90% of the time.

Reply:
This was a scenario to get from A to B in explaining how to use multiple env files. Id be super interested if you have a leaner, and by all accounts better example 😊